Caltech-IPAC/firefly
# DM-10560: clean of image retrieval error messages

DM-10560: clean of image retrieval error messages

  - Error message are more accurate and short
  - The right message is passed though exception, instead of have a generic message
  - http codes are translated into messages for failure cases
  - removed 2 old Exception classes

_To Test:_

 - Try msx image with target m81
 - Try sdss image with target m31
 - Take your computer offline an try a wise image or a dss image
 - Load http://localhost:8080/firefly/demo/ffapi-highlevel-test.html notice the error message on one of the plots
 - There are many more error messages but they are hard to test when the servers are up
